src/session/session_api.c

```diff
@@ -813,7 +813,7 @@ __session_log_printf(WT_SESSION *wt_session, const char *fmt, ...)
     va_list ap;
 
     session = (WT_SESSION_IMPL *)wt_session;
-    SESSION_API_CALL_PREPARE_NOT_ALLOWED_NOCONF(session, log_printf);
+    SESSION_API_CALL_PREPARE_ALLOWED_NOCONF(session, log_printf);
```

Comment:
This change is breaking some of the unit tests.

Reply:
Thanks, fixed in 7cf79d0.